### DISCUSSION TITLE
Having a dedicated button to delay the transition from loading a map to gameplay

### DISCUSSION THREAD

**Discussion by nathannino** [ppy/osu#13130] [Ideas]
**Describe the new feature:** 
This button will allow you to take a break or do last minute preparation while you are loading the map. 
There is already a workaround to have this feature in current releases, as opening the option menu does exactly what I want this button to do. However, when restarting a map, the toolbar doesn't reappear on the loading screen. You could still open them with the shortcut, but some users, and I assume this mainly affect mobile users, will not have a keyboard on hand.
**Proposal designs of the feature:** 
Having a button on the loading screen that, when active, will prevent the transition to gameplay, while also notifying the player on why the client isn't switching to gameplay.

**Comment by enoslayd** [comment]
Like a "tap to start"?

**Comment by peppy** [comment]
Is the issue here that load times are longer than the transition? If so, I think we should hold off until we fix that part (to the point the transition is a constant length on 99% of beatmaps).

**Comment by nathannino** [comment]
> Is the issue here that load times are longer than the transition? If so, I think we should hold off until we fix that part

Not sure if I completely understood what you are saying here, but I don't know what you are mentioning. However, I recognize that I did throw some words around without being sure that there were used correctly, and it's completely possible that I used the wrong keyword to search the issue you are talking about or that it isn't on GitHub yet. Also, while taking a second look at the issue list to find what you mentioned, I have found that #9039 could remove what I would consider being the biggest reason for this feature request: being able to take a short break without the need to open a menu or leave the loading screen

> Like a "tap to start"?

Yeah, but that you have to tap to enable it. As such, if you don't tap, it won't wait for you to tap it again to start

**Comment by peppy** [comment]
Let me rephrase: If the "load" screen was always there for exactly half a second regardless of beatmap, would you still need this button?

**Comment by nathannino** [comment]
Yes. My main use for it would be to have time to move stuff on my desk in between plays without having to hold the menu button or remove the delay on the quick retry key (since the menu key and the quick retry key shares the same purpose), so as long as it goes to that screen in between plays, I would find it useful
(However, I still want to point out that opening the option screen has the same effect, but needs a keyboard input in between retries and completely removing the need for the loading screen on retries would remove the usefulness of the button)

**Comment by peppy** [comment]
Have you considered using the pause screen instead of quick retry? The purpose of quick retry is to retry, quickly.

**Comment by nathannino** [comment]
Honestly, it's based on habits from when having your mouse on the toolbar and the visual settings would also pause it, and since my retry key is so close to my clicking keys, I don't want to remove the cooldown on the menu button. As such, I personally find it easier to simply quick retry and do my stuff than pausing normally. Finally, pausing gameplay is, well, interupting gameplay while pausing in the loading screen is not, so even though both are the same on paper, I personally feel better doing it outside gameplay.

Still, I completely aggree on the fact that this feature is not neccessary, but I still wanted to suggest it, since I have been enjoying using the various workarounds that makes it work in previous and current versions and having it become an official feature would, I hope, remove the requirement for the keyboard, making it easier to use.

In conclusion, while I do believe that it's a cool feature, and I would like it to become an official feature, I cannot defend the neccessity of it, and if it does go against the current plans for the game, there isn't really a big argument to defend it, except for the fact that I think that it feels better.
